### ISSUE TITLE
Win32 Exception When Enumerating Sessions in a Down State

### ISSUE THREAD

**Issue by GoogleCodeExporter** [danports/cassia#36]
```
What steps will reproduce the problem?

1.  Terminal server has a session in the Down state, typically due to a 
hung user process or a hung thread in csrss.exe

2.Call the GetSessions method of an ITerminalServer object


What is the expected output? What do you see instead?

Sessions should be enumerated;  a Win32 Exception occurs with the message 
'A close operation is pending on the session'


What version of the product are you using? On what operating system?

Version 2.0.0.60 on Windows Server 2003 SP2 (both 32 and 64bit)


Please provide any additional information below.


```

Original issue reported on code.google.com by `clokenda...@gmail.com` on 27 May 2010 at 10:12
- Merged into: #24
